### ISSUE TITLE
[Flaky Test] Higher-Level Review Add / Remove Issues page Veteran with future ratings

### ISSUE THREAD

**Issue by ferristseng** [department-of-veterans-affairs/caseflow#13459]
## Background/context/resources

```
  1) Higher-Level Review Add / Remove Issues page Veteran with future ratings when show_future_ratings featuretoggle is enabled 
     Failure/Error: expect(higher_level_review.receipt_date).to eq(receipt_date)

       expected: Sat, 25 May 2019
            got: Tue, 30 Apr 2019

       (compared using ==)

       Diff:
       @@ -1,2 +1,2 @@
       -Sat, 25 May 2019
       +Tue, 30 Apr 2019
     # ./spec/feature/intake/higher_level_review_spec.rb:701:in `block (4 levels) in <top (required)>'
```

 - Circle CI Error: https://app.circleci.com/jobs/github/department-of-veterans-affairs/caseflow/102457/parallel-runs/4/steps/4-120
 - Has the test already been skipped in the code?
   - [ ] Skipped
   - [x] Not Skipped
 - Related Flakes
    + 

## Approach

Time box this investigation and fix.

Remember that if a test has been skipped for a decent amount of time, it may no longer map to the exact code.

If you reach the end of your time box and don't feel like the solution is in sight:
  - [ ] document the work you've done, including dead ends and research
  - [ ] skip the test in the code
  - [ ] file a follow on ticket
  - [ ] close this issue

**Comment by pkarman** [comment]
fix for this in #13448

**Comment by pkarman** [comment]
fixed pulled into #13567
